docs/CONTRIBUTING.md

```diff
@@ -62,15 +87,39 @@ Example, should we add `fasttemplate`
 
 No, we should not add that dependency.
 
-### Test Policy
+#### Test Policy
 
 Changes without either unit or e2e tests are unlikely to be accepted.
 See [the pull request template](https://github.com/argoproj/argo-workflows/blob/main/.github/pull_request_template.md).
 
-### Contributor Workshop
+### Other Contributions
 
-Please check out the following resources if you are interested in contributing:
+* [Reviewing PRs](#reviewing-prs)
+* Responding to questions in the [Slack](#slack) channels
+* Responding to questions in [Github Discussions](https://github.com/argoproj/argo-workflows/discussions)
+* [Triaging new bugs](#triaging-bugs)
+
+#### Reviewing PRs
+
+Anybody can review a PR; however, those who are in the roles of Reviewer and above are expected to assign themselves to PRs.
+Since Reviewers cannot merge PRs, then once they have approved a PR, they should request a review from one or more Approvers/Leads.
```

Comment:
```suggestion
Anybody can review a PR.
If you are in a [designated role](#roles), add yourself as an "Assignee" to a PR if you plan to lead the review.
If you are a Reviewer or below, then once you have approved a PR, request a review from one or more Approvers and above.
```

- address the reader as "you", per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#address-the-reader-as-you)
  - also matches the existing verbiage of this doc
  - also made the tone gentler as the Contributing Guide is giving directions, which is a different tone from "you are expected to" (which is more accurate for the incentive system, which has specific requirements)
  
- simplify the wording a bit (e.g. removed "you should"), per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)
  
- linked to "Roles" section as specific roles are referenced here
  
- changed it to Reviewer and below; I certainly led review of various PRs as a Member and we have other contributors and Members who do so occasionally as well. if they're doing it already and can assign themselves, then why not do so I figure